Jenkinsfile

```
@@ -151,27 +151,9 @@ builds.ath = {
     node('docker-highmem') {
       // Just to be safe
       deleteDir()
-      def fileUri
-      def metadataPath
-      dir('sources') {
-        checkout scm
-        def mavenOptions = [
-          '-Pquick-build',
-          '-Dmaven.repo.local=$WORKSPACE_TMP/m2repo',
-          '-am',
-          '-pl',
-          'war',
-          'package',
-        ]
-        infra.runMaven(mavenOptions, 11)
-        dir('war/target') {
-          fileUri = 'file://' + pwd() + '/jenkins.war'
-        }
-        metadataPath = pwd() + '/essentials.yml'
-      }
-      dir('ath') {
-        runATH jenkins: fileUri, metadataFile: metadataPath
-      }
+      checkout scm
+      sh './ath.sh'
```

```suggestion
      sh 'bash ./ath.sh'
```

Calling bash explicitly ensures that the script `ath.sh`is not required to have the "execution" bit for the current user.